docs/navigate.md

```diff
@@ -57,6 +59,20 @@ update itself while the user is panning around on it (set to null to disable
 intermediate pans, the plot will then not update until the mouse button is
 released).
 
+**mode** a string specifies the pan mode for mouse interaction. Accepted values:
+empty string for normal mode: no pan hint or direction snapping;
```

Comment:
This is still not a good idea for identifying one of this modes. We need to give it a string value, and treat this property as if it were an enum. One term that came up, that I am fine with, is 'manual'.